### DISCUSSION TITLE
Attachments may cause row not to be displayed in a table.

### DISCUSSION THREAD

**Discussion by magickaito** [Budibase/budibase#6101] [Help] [answered]
My app is stuck in an error state. For any tables, rows with image attachments will be hidden.
I noticed this issue was raised before but closed without a fix: https://github.com/Budibase/budibase/issues/2892

Unfortunately I cannot reproduce this in a new app, but it is happening in my existing app.
So to reproduce, I stripped down my app to a bare minimum, removed all tables, all automation, all screen, I can still see the problem. You can simply import this minimal app to see the problem:

- Create new app and import the attached text file.
- Check out the "Debug" table. You will only see 1 row.
- Click export to CSV, you will see 3 rows exported in csv. 2 hidden rows with image attachments.
- Add new rows with image attachment, they will be hidden
- Add new rows without image attachment, they will show up properly
- Create new table, and it will be the same.
[Debug Attachment Row-export-1653646675008.txt](https://github.com/Budibase/budibase/files/8785936/Debug.Attachment.Row-export-1653646675008.txt)

**Comment by melohagan** [comment, marked as answer]
Hi @magickaito 

There may be a bug in your app export. I wonder if you did a large version jump maybe?

I was able to fix it by resolving a conflict in the **_design/database** in CouchDB.

Steps I took; although not sure if this can be done in cloud. 
1. Imported your app
2. Opened the CouchDB UI `localhost:4005/_utils` username & password = budibase
3. Navigate to design database for the imported app `http://localhost:4005/_utils/#database/app_dev_ad3020dd9dab4bd1ad40fe0c91e57f63/_design/database`
4. Clicked on the Conflicts tab
![Screenshot 2022-05-30 at 10 24 58](https://user-images.githubusercontent.com/101575380/170961598-c546ef8c-6736-4b15-8666-75f585fc72ad.png)
5. Select as winner
![Screenshot 2022-05-30 at 10 25 35](https://user-images.githubusercontent.com/101575380/170961703-ef453913-2c8f-4938-b2a4-38c5e90e8c56.png)
 
Rows are now showing:
![Screenshot 2022-05-30 at 10 26 17](https://user-images.githubusercontent.com/101575380/170961861-b574518b-56f0-4ddf-8de6-f895a9b64284.png)

@mike12345567 Do you have any further thoughts on this?

**Reply by magickaito** [reply]
I haven't tried self-hosted, I will give it a try to see if I can avoid re-creating my app from scratch. 
p/s: What do u mean large version jump? Is it a self-hosted feature? I don't think I did that... :D

**Reply by melohagan** [reply]
Hey @magickaito I just meant maybe the app was created with a very old version and was upgraded recently, but it's probably not that - just a spitball. 

That's a good idea to import and fix locally, then reimport in the cloud!

**Reply by magickaito** [reply]
@melohagan I followed the steps to resolve conflicts on digitalocean, and my project is working with attachment again. thanks!

**Reply by melohagan** [reply]
Awesome, good stuff :)
